src/components/wallet/PageTransactions.vue

```diff
@@ -25,10 +25,13 @@
           Looking for older transactions?
         </div>
         <div slot="subtitle">
-          Unfortunately, we can't display transactions from previous chains
-          right now.
-          <a class="intercom-button" @click="handleIntercom()">Let us know</a>
-          if you'd like access these transactions.
+          <p>
+            Lunie cannot display transactions from previous chains in your activity page. 
+          </p>
+          <p>if you would like to view information from previous chain upgrades
```

Comment:
I try to follow this style guide for links: https://polaris.shopify.com/content/actionable-language#section-links